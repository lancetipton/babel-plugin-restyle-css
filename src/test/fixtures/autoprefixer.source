reStyle`.foo { animation-name: ${foo}; color: ${ bar } }`;

