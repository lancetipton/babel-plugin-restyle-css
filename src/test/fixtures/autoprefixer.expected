({
  ".foo": {
    webkitAnimationName: foo,
    animationName: foo,
    color: bar
  }
});